laravel/resources/views/layouts/nav-main.blade.php

```diff
@@ -0,0 +1,88 @@
+                    <a class="nav-link {{ (str_starts_with(Route::currentRouteName(), 'instance')) ? 'active' : '' }}" aria-current="page" href="{{ Route('instances') }}">Instances</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link {{ (str_starts_with(Route::currentRouteName(), 'template')) ? 'active' : '' }}" aria-current="page" href="{{ Route('templates') }}">Templates</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link {{ (str_starts_with(Route::currentRouteName(), 'banner')) ? 'active' : '' }}" aria-current="page" href="{{ Route('banners') }}">Banners</a>
+                </li>
+            </ul>
+        </div>
+        <div class="collapse navbar-collapse justify-content-end text-bg-dark">
+            <ul class="navbar-nav my-auto">
+                <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle
+                    {{ (str_starts_with(Route::currentRouteName(), 'administration.')) ? 'active' : '' }}"
+                       href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                        Administration
+                    </a>
+                    <ul class="dropdown-menu">
+                        @can('view users')
+                        <li><a class="dropdown-item {{ (Route::currentRouteName() == 'administration.users') ? 'active' : '' }}" href="{{Route('administration.users')}}">User</a></li>
+                        @endcan
+                        @can('view roles')
+                        <li><a class="dropdown-item {{ (Route::currentRouteName() == 'administration.roles') ? 'active' : '' }}" href="{{ route('administration.roles') }}">Roles</a></li>
+                        @endcan
+                        @can('view fonts')
+                        <li><a class="dropdown-item {{ (Route::currentRouteName() == 'administration.fonts') ? 'active' : '' }}" href="{{ route('administration.fonts') }}">Fonts</a></li>
+                        @endcan
+                        @can('view system status')
+                        <li><a class="dropdown-item {{ (Route::currentRouteName() == 'administration.systemstatus') ? 'active' : '' }}" href="{{ route('administration.systemstatus') }}">System Status</a></li>
+                        @endcan
+                        @can('view php info')
+                        <li><a class="dropdown-item {{ (Route::currentRouteName() == 'administration.phpinfo') ? 'active' : '' }}" href="{{ route('administration.phpinfo') }}">PHP Info</a></li>
+                        @endcan
+                    </ul>
+                </li>
+            </ul>
+            <ul class="navbar-nav my-auto">
+                <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle {{ (request()->is('profile*') ? 'active' : '') }}" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
```

Comment:
[SUGGESTION]

To have the same code logic everywhere:
```suggestion
                    <a class="nav-link dropdown-toggle {{ (str_starts_with(Route::currentRouteName(), 'profile.')) ? 'active' : '') }}" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
```